spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -227,13 +227,15 @@ def env(self):
             env_vars['SPY_RUN_CYTHON'] = True
 
         # App considerations
+        # ??? Do we need this?
         if (running_in_mac_app() or is_pynsist()):
             if default_interpreter:
                 # See spyder-ide/spyder#16927
                 # See spyder-ide/spyder#16828
                 # See spyder-ide/spyder#17552
                 env_vars['PYDEVD_DISABLE_FILE_VALIDATION'] = 1
             else:
+                # ??? Do we need this?
                 env_vars.pop('PYTHONHOME', None)
```

Comment:
I'm leaving notes for followup PR(s). But think you are correct.

Reply:
We should probably leave it to prevent users from overriding the default place where Python looks for the standard library:

https://www.geeksforgeeks.org/environment-variables-in-python/